src/Compilers/Core/Portable/Symbols/Attributes/CommonTypeEarlyWellKnownAttributeData.cs

```diff
@@ -113,5 +113,23 @@ public bool HasCodeAnalysisEmbeddedAttribute
             }
         }
         #endregion
+
+        #region InterpolatedStringHandlerAttribute
+        private bool _hasInterpolatedStringHandlerAttribute;
+        public bool HasInterpolatedStringHandlerAttribute
```

Comment:
>HasInterpolatedStringHandlerAttribute [](http://example.com/codeflow?start=20&length=37)

Should this be moved to C# specific type?

Reply:
I'll have to add one as we don't have one for C# currently, but that's doable.